### PR TITLE
Erik update react native

### PR DESCRIPTION
**Pull request by AirTechWick**
Added running an SDK 1.2 version of the demo in the guide. Can try and implement get options in the future.

### CODE REVIEW

File: docs/react-native-theta-client-1_2.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+comments: true
+---
+
+# React Native Theta Client v1.2 Walkthrough
```
Comment (jcasman) [inline docs/react-native-theta-client-1_2.md:5]: Please make all "THETA" mentions all caps. It's RICOH THETA.

File: docs/react-native-theta-client-1_2.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Overview
+
+Walks through [Theta Client v1.2](https://github.com/ricohapi/theta-client/releases/tag/1.2.0) and tests out the new features in React Native.
```
Comment (jcasman) [inline docs/react-native-theta-client-1_2.md:9]: Please fix

**Review by jcasman — CHANGES_REQUESTED** [review]
Please fix the all caps spelling.

Please add a link to this information: https://community.theta360.guide/t/theta-client-official-sdk-v1-2-released-with-client-mode-time-shift-previewformat-and-more/9067

I can't use yarn on either Mac or Windows. Did you have any issues using yarn? Are you able to do a yarn installation section?

**Comment by AirTechWick** [comment]
In this commit shown above 136e32e I also fixed the Capitalization

**Pull request merged**